test/jtreg-ext/requires/VMProps.java

```diff
@@ -291,6 +292,28 @@ protected void vmGC(SafeMap map) {
         }
     }
 
+    /**
+     * "jtreg -vmoptions:-Dtest.cds.runtime.options=..." can be used to specify
+     * the GC type to be used when running with a CDS archive. Set "vm.gc" accordingly,
+     * so that tests that need to explicitly choose the GC type can be excluded
+     * with "@requires vm.gc == null".
+     *
+     * @param map - property-value pairs
+     */
+    protected void vmGCforCDS(SafeMap map) {
```

Comment:
as this might impact any tests which use `vm.gc` and not just CDS tests, I think it would be better to first check if `vm.gc` is(will be) set to non-null value. given it's done by jtreg in another class and using another instance of Map, we can't just call `Map::contains` and have to check if GC has been explicitly selected by smth like `GC.selected().isSelectedErgonomically()`.

it also might be a good idea to comment at L#124 to indicate that this method overrides `vm.gc` value.

Reply:
Thanks for the review. I've fixed it as you suggested.